src/imp/tracer_imp.js

```diff
@@ -680,6 +680,8 @@ export default class TracerImp extends EventEmitter {
     }
 
     _buffersAreEmpty() {
+        console.log("_buffersAreEmpty runtimeGUID", this._runtimeGUID);
```

Comment:
I may be due to the unit-only code path where the runtime GUID is forced to be set: https://github.com/lightstep/lightstep-tracer-javascript/blob/master/test/util/util.js#L12

This was done to circumvent the code that otherwise early outs if the Tracer is not fully configured for "normal" reporting.